examples/assetmanagement/asset_utilization.py

```diff
@@ -0,0 +1,123 @@
+import threading
+import time
+from datetime import datetime
+from uuid import uuid4
+
+from nisystemlink.clients.assetmanagement import AssetManagementClient
+from nisystemlink.clients.assetmanagement.models import (
+    AssetBusType,
+    AssetIdentification,
+    StartUtilizationRequest,
+)
+from nisystemlink.clients.core._http_configuration import HttpConfiguration
```

Comment:
You shouldn't need to import anything with an underscore. `nisystemlink.clients.core.__init__` imports `HttpConfiguration` for you.

```suggestion
from nisystemlink.clients.core import HttpConfiguration
```

Reply:
Updated.